diff --git a/tools/date.c b/tools/date.c
index 13da804..e1988dc 100644
--- a/tools/date.c
+++ b/tools/date.c
@@ -564,7 +564,7 @@ day_number(date_t *d)
  * TODO: None
  */
 int
-getDateWeightFromJulian(jDay, nDistribution)
+getDateWeightFromJulian(int jDay, int nDistribution)
 {
 	date_t dTemp;
 	int nDay;
diff --git a/tools/tdef_functions.c b/tools/tdef_functions.c
index 2e902d6..e6afd02 100644
--- a/tools/tdef_functions.c
+++ b/tools/tdef_functions.c
@@ -171,7 +171,7 @@ table_func_t s_tdef_funcs[] = {
 };
 
 table_func_t *
-getTdefFunctionsByNumber(nTable)
+getTdefFunctionsByNumber(int nTable)
 {
    if (nTable >= S_BRAND)
       return(&s_tdef_funcs[nTable - S_BRAND]);
diff --git a/tools/tdefs.c b/tools/tdefs.c
index fe8cc47..7a07153 100644
--- a/tools/tdefs.c
+++ b/tools/tdefs.c
@@ -147,7 +147,7 @@ getTdefsByNumber(int nTable)
 }
 */
 tdef *
-getSimpleTdefsByNumber(nTable)
+getSimpleTdefsByNumber(int nTable)
 {
    if (nTable >= S_BRAND)
       return(&s_tdefs[nTable - S_BRAND]);
